src/servers/cron.ts

```diff
@@ -48,6 +49,13 @@ const main = async () => {
     if (result instanceof Error) throw result
   }
 
+  const swapOutJob = async () => {
+    const amount = toSats(getSwapConfig().swapOutAmount)
```

Comment:
I think `getSwapConfig().swapOutAmount` should already returning the correct type